### PR TITLE
LaPy Test Cases

### PR DESCRIPTION
**Pull request by engrosamaali91**

### CODE REVIEW

**Comment by engrosamaali91** [comment]
In this commit [de75fc1](https://github.com/Deep-MI/LaPy/pull/43/commits/de75fc1aa3b73d33095c424f4c11390d543e8ff0) a test file and expected_outcomes.json files have been removed from lapy directory.

File: lapy/utils/tests/test_tet_mesh.py [hunk 1]
```diff
@@ -211,7 +203,7 @@ def test_correct_orientations_and_boundary(tet_mesh_fixture):
 
 def test_boundary_surface_volume(tet_mesh_fixture):
     """
-    Testing computation of volume for the boundary surface mesh
+    Test orientation correction and boundary surface orientation.
```
Comment (m-reuter) [inline lapy/utils/tests/test_tet_mesh.py:206]: it this correct? the function name indicates that volume is tested not orientation?

File: lapy/utils/tests/expected_outcomes.json [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            "tolerance": 1e-4
+        },
+        "test_compute_distance_tet": {"exp_compute_distance": 0.0},
+        "test_triangle_mesh_visualization_using_checksum": {
+            "real_img_checksum": "abaaa757a1e5c1f42a8302fbfe068f49"
+        },
+        "test_TriaMesh_Geodesics": {
+            "real_img_checksum": "8166f546e6630f017d3ec1f655c0ac85"
+        },
+        "test_Laplase_Geodesics": {
```
Comment (m-reuter) [inline lapy/utils/tests/expected_outcomes.json:125]: Laplace is written with a "c"

**Comment by engrosamaali91** [comment]
Hello, All checks have passed for committed tests [3bc26d3](https://github.com/Deep-MI/LaPy/pull/43/commits/3bc26d3d512c6f2c58000c4b40b094f60505114a), removing the flag from work in progress.
For remaining tests i.e visualization and Triamesh_Geodesics only I would like to create a new branch and a new PR.

**Comment by m-reuter** [comment]
I think the Laplase typo is still not fixed? Also there is some code in the test scripts that is not part of any test functions?

**Comment by engrosamaali91** [comment]
> I think the Laplase typo is still not fixed? Also there is some code in the test scripts that is not part of any test functions?

In this commit [49a96e4](https://github.com/Deep-MI/LaPy/pull/43/commits/49a96e42e5478004917b1b1fcd1c84cc10969a3e) I have adjusted the code with test functions, the code was still part of the test functions and it was computing variables globally. 

Now it is directly being computed from functions itself.

Laplase typo is fixed.

**Pull request merged**